### PR TITLE
adds CentOS 7.

### PR DESCRIPTION
**Pull request by danvaida**
See relevant build [here](https://hub.docker.com/r/danvaida/ansible-docker-images/builds/brwrdvthxmvcatcvqintcrg/).

### CODE REVIEW

File: centos/7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+FROM centos:7
+
+LABEL maintainer me@danvaida.com
+
+ENV container docker
+
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
```
Comment (schrodervictor) [inline centos/7/Dockerfile]: I would definitely try to break this line into multiple lines

File: centos/7/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+RUN yum update -y && \
+    yum install -y epel-release
+RUN yum install -y ca-certificates \
+                   gcc \
+                   openssl \
+                   openssl-devel \
+                   python2-pip \
+                   python-devel \
+                   libffi-devel
```
Comment (schrodervictor) [inline centos/7/Dockerfile]: Won't we pin the versions? I also don't see the cleanup of the build dependencies, like you did for the Jessie image.

File: centos/7/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+    rm -f /lib/systemd/system/basic.target.wants/*;\
+    rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+VOLUME [ "/sys/fs/cgroup" ]
+
+RUN echo 'root:root' | chpasswd
+
+RUN yum update -y && \
+    yum install -y epel-release
```
Comment (schrodervictor) [inline centos/7/Dockerfile]: It's missing a line break between lines 21 and 22...

Comment (danvaida) [inline centos/7/Dockerfile]: not really. there's a new install command on line 22. or are you asking why am I not installing everything at once?

File: centos/7/Dockerfile (hunk 2 above)
Comment (danvaida) [inline centos/7/Dockerfile]: Before & after cleanup:
```bash
$ docker images | grep -i centos
centos7_cleanup                  latest              cae890d3868b        8 seconds ago       286 MB
ansible-roles-test-centos7       latest              54560168c55d        4 days ago          501 MB
```

**Comment by schrodervictor** [comment]
Awesome! Approved.

**Pull request merged**